src/Core/src/Platform/Windows/ShadowExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿#nullable disable
+
+using System;
+using System.Diagnostics;
+using System.Numerics;
+using System.Threading.Tasks;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Hosting;
+using Microsoft.UI.Xaml.Media.Imaging;
+using Windows.Foundation;
+using Windows.Graphics.Imaging;
+
+
+namespace Microsoft.Maui
+{
+	public static class ShadowExtensions
+	{
+
+		public static async Task<CompositionBrush> GetAlphaMaskAsync(this UIElement element)
+		{
+			CompositionBrush mask = null;
+
+			try
+			{
+				if (element is TextBlock textElement)
+				{
+					mask = textElement.GetAlphaMask();
```

Comment:
For some reason, using a Label (in WinUI a TextBlock) and getting the AlphaMask generate a shadow with a size more smaller than the control size. not a PR blocker but we can include a comment to review that.

Reply:
done